src/PerformanceTests/NServiceBus5/App.config

```diff
@@ -1,26 +1,49 @@
 <configuration>
   <configSections>
     <section name="MessageForwardingInCaseOfFaultConfig" type="NServiceBus.Config.MessageForwardingInCaseOfFaultConfig, NServiceBus.Core" />
     <section name="AuditConfig" type="NServiceBus.Config.AuditConfig, NServiceBus.Core" />
     <section name="RijndaelEncryptionServiceConfig" type="NServiceBus.Config.RijndaelEncryptionServiceConfig, NServiceBus.Core" />
     <section name="Logging" type="NServiceBus.Config.Logging, NServiceBus.Core" />
-    <section name="nlog" type="NLog.Config.ConfigSectionHandler, NLog" />
+    <section name="log4net" type="log4net.Config.Log4NetConfigurationSectionHandler, log4net" />
   </configSections>
 
-  <nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-    <targets async="true">
-      <target name="file" xsi:type="File" fileName="trace.log" layout="${longdate:universalTime=true}|${level:uppercase=true}|${threadid:padding=2}|${logger}|${message}${onexception:${newline}${exception:format=tostring}}" />
-      <target name="trace" xsi:type="OutputDebugString" layout="${level:uppercase=true}|${threadid:padding=2}|${logger}|${message}${onexception:${newline}${exception:format=tostring}}" />
-      <target name="console" xsi:type="ColoredConsole" layout="${longdate}|${level:uppercase=true}|${threadid:padding=2}|${logger}|${message}${onexception:${exception:format=message}}" />
-    </targets>
-    <rules>
-      <logger name="NServiceBus.Azure.Transports.WindowsAzureServiceBus.AzureServiceBusQueueCreator" maxlevel="Info" final="true" />
-      <logger name="NServiceBus.Distributor.MSMQ.*" maxlevel="Info" final="true" />
-      <logger name="NServiceBus.RecoverabilityExecutor" maxlevel="Info" final="true" />
-      <logger name="*" minlevel="Debug" writeTo="file,trace,console" />
-    </rules>
-  </nlog>
+  <log4net>
+    <appender name="Console" type="log4net.Appender.ColoredConsoleAppender" >
+      <mapping>
+        <level value="ERROR" />
+        <foreColor value="Yellow" />
+      </mapping>
+      <mapping>
+        <level value="WARN" />
+        <foreColor value="Purple" />
+      </mapping>
+      <mapping>
+        <level value="FATAL" />
+        <foreColor value="RED" />
+      </mapping>
+      <layout type="log4net.Layout.PatternLayout">
+        <conversionPattern value="%date|%level|%thread|%logger|%message%newline%exception" />
+      </layout>
+    </appender>
+    <appender name="File" type="log4net.Appender.FileAppender" >
+      <file value="trace.log" />
+      <layout type="log4net.Layout.PatternLayout">
+        <conversionPattern value="%date|%level|%thread|%logger|%message%newline%exception" />
+      </layout>
+    </appender>
+    <appender name="Trace" type="log4net.Appender.TraceAppender" >
+      <layout type="log4net.Layout.PatternLayout">
+        <conversionPattern value="%date|%level|%thread|%logger|%message%newline%exception" />
+      </layout>
+    </appender>
+    <root>
```

Comment:
I do not think the current log4net configuration is doing asynchronous logging.

Reply:
can you point me towards the log4net docs about asynchronous logging?

Reply:
Just searched for it... but log4net doesn't have an async appender OOTB, which sucks. Just leave it as is then. If I run the performance tests and see a noticeable performance dip then we can add an async appender.